SchedulerJobs/SchedulerJobs.Services/AnonymiseHearingsConferencesDataService.cs

```diff
@@ -34,12 +39,27 @@ private async Task DeleteUserAsync()
             // get users that do not have hearings in the future and have had hearing more than 3 months in the past. 
             // (exclude judges, vhos, test users, performance test users.
             var usersToDelete = await _bookingsApiService.GetUsersWithClosedConferencesAsync();
-            if (usersToDelete != null && usersToDelete.Usernames != null)
+            
+            _logger.LogInformation($"AnonymiseHearingsConferencesDataService: Found {usersToDelete.Usernames.Count} users to delete");
+            
+            if (usersToDelete?.Usernames != null)
             {
+                var deletedCount = 0;
                 foreach (var username in usersToDelete.Usernames)
                 {
-                    await _userApiService.DeleteUserAsync(username);
+                    try
+                    {
+                        await _userApiService.DeleteUserAsync(username);
+                        deletedCount++;
+                        _logger.LogInformation($"AnonymiseHearingsConferencesDataService: deleted {username}");
+                    }
+                    catch (Exception e)
+                    {
+                        _logger.LogError($"AnonymiseHearingsConferencesDataService: failed to delete {username}");
```

Comment:
please pass the exception to the logger here, would be good to understand why it failed too. If the exception is UserApiException it will contain the error message